src/site/xdoc/server/config-vault.xml

```diff
@@ -0,0 +1,49 @@
+<?xml version="1.0"?>
+<!--
+  Licensed to the Apache Software Foundation (ASF) under one
+  or more contributor license agreements.  See the NOTICE file
+  distributed with this work for additional information
+  regarding copyright ownership.  The ASF licenses this file
+  to you under the Apache License, Version 2.0 (the
+  "License"); you may not use this file except in compliance
+  with the License.  You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing,
+  software distributed under the License is distributed on an
+  "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+  KIND, either express or implied.  See the License for the
+  specific language governing permissions and limitations
+  under the License.    
+-->
+<document>
+
+    <properties>
+        <title>Apache James Server 3 - Deleted Messages Vault Configuration</title>
+    </properties>
+
+    <body>
+
+        <section name="Deleted Messages Vault Configuration">
+
+            <p>Note: This feature requires Guice wiring</p>
+
+            <subsection name="deletedMessageVault.properties">
+
+                <p>Consult <a href="https://github.com/apache/james-project/blob/master/dockerfiles/run/guice/cassandra/destination/conf/deletedMessageVault.properties">deletedMessageVault.properties</a>
+                    to get some examples and hints.</p>
+                <br/>
+                <dl>
+                    <dt><strong>urlPrefix</strong></dt>
+                    <dd>Define prefix value of namespace in MailRepository where deleted messages are stored (default: default mail repository protocol + var/deletedMessages/vault)</dd>
+
+                    <dt><strong>retentionPeriod</strong></dt>
+                    <dd>Deleted messages stored in the Deleted Messages Vault are expired after this period, (default: 1 year)</dd>
+                </dl>
+
+            </subsection>
```

Comment:
Maybe I would add a subsection that you need to configure the pre-deletion hooks as well for the vault to be fully enabled, with probably a redirection to the `config-listeners.xml` section for its configuration. Wdyt?

Reply:
You did put a thumbs up but I didn't see the change, why mark it as resolved? :)

Reply:
My bad I didn't see :)